click-to-load/click_to_load_config_reference.json

```diff
@@ -0,0 +1,50 @@
+{
+    "Tracker INC": {
+        "domains": [
+            "tracker.test",
+            "tracker.net"
+        ],
+        "excludedSubdomains": [
+            "excluded.tracker.test"
+        ],
+        "excludedDomains": [
+            {
+                "domain": "local-exception.test",
+                "reason": "Local exception"
+            }
+        ],
+        "surrogates": [
+            {
+                "rule": "tracker\\.test\\/[a-z_A-Z]+\\/sdk.js",
+                "surrogate": "surrogate.js",
+                "xray": "surrogate-xray.js"
+            }
+        ],
+        "informationalModal": {
+            "icon": "blocked_logo.svg",
+            "messageTitle": "Logging in with Tracker lets them track you",
+            "messageBody": "Once you're logged in, DuckDuckGo can't block Tracker content from tracking you on this site.",
+            "confirmButtonText": "Log In",
+            "rejectButtonText": "Go back"
+        },
+        "clicksBeforeSimpleVersion": 3,
+        "elementData": {
+            "Page iFrames": {
+                "selectors": [
+                    "iframe[src*='://www.tracker.com/plugins/page']",
+                    "iframe[src*='://www.tracker.com/v2.0/plugins/page']"
+                ],
+                "replaceSettings": {
```

Comment:
I wonder if we should have tests for the `loginButton` and `blank` replacement types as well?

Reply:
Tests are focused on request blocking rather than in-HTML replacements, do you think including loginButton and blank would still be useful?

Reply:
Ah right, no I don't think including those replacement types would test any more of the request blocking code-paths in the background page.